utils/parachain-node/runtime/src/lib.rs

```diff
@@ -722,6 +722,8 @@ impl pallet_ibc::Config for Runtime {
 	type Whitelist = AllowAll;
 	type MemoMessage = MemoMessage;
 	type HandleMemo = ();
+	type PoolId;
+	type Pablo; // TODO: vim - can I get help to construct an impl of this? same for mock
```

Comment:
@vimukthi-git

Reply:
You can configure a Pablo instance for the runtime same as in the dex-router here https://github.com/ComposableFi/composable/blob/main/code/parachain/frame/dex-router/src/mock.rs#L238